app/packs/stylesheets/decidim/decidim_awesome/awesome_application.scss

```
@@ -2,6 +2,7 @@
 @import "stylesheets/decidim/decidim_awesome/editors/markdown_editor";
 @import "stylesheets/decidim/decidim_awesome/editors/quill_editor";
 @import "stylesheets/decidim/decidim_awesome/forms/autosave";
+@import "stylesheets/decidim/decidim_awesome/vote/vote";
```

```suggestion
@import "stylesheets/decidim/decidim_awesome/voting/three_flags";
```
Better be more specific, in case we add more systems